src/Components/SelectedCountryDetailsComponent/SelectedCountryDetailsComponent.js

```diff
@@ -4,32 +4,22 @@ import RaisedButton from 'material-ui/RaisedButton';
 
 import './SelectedCountryDetailsComponent.css';
 
-const style = {
-    height: 50,
-    width: 300,
-    paddingTop: 25,
-    margin: 10,
-    textAlign: 'center',
-    display: 'inline-block',
-    verticalAlign: 'middle',
-    lineHeight: 'normal'
-};
-
-
 export const SelectedCountryDetailsComponent = (props) => {
-
     return (<div>
         <Paper>
             <p>Name: {props.details.name}</p>
             <p>Capital: {props.details.capital}</p>
         </Paper>
 
         {props.details.borders.map((border) => {
-            // return (<Paper style={style} zdepth={5} key={border}>{border}</Paper>)
-
 
             return (
-                    <RaisedButton key = {border} label= {border} className='raised-button' />
+                    <RaisedButton
+                        key = {border}
+                        label= {border}
+                        className='b-button_borders'
+                        onClick={props.changeCountryButtonClick.bind(this,border)}
```

Comment:
nie robimy bind w jsx, to powinno być wyniesione poza dummy component i na `onClick` dajemy sam handler

`() => handleOnClick(border)`

Reply:
Mam z tym problem. Omwowmy to dzisiaj.